### PR TITLE
feat: integrate fortmatic provider

### PR DESCRIPTION
**Pull request by hanszhoou**
closes #

### CODE REVIEW

**Comment by Tedko** [comment]
just fyi we can move the feat into next or next next release

File: packages/mask/src/plugins/EVM/UI/components/InjectedProviderBridge.tsx [hunk 1]
```diff
@@ -10,27 +10,48 @@ export interface InjectedProviderBridgeProps {}
...
+                        params: payload.params,
+                    })
+                } catch (error: unknown) {
+                    EVM_Messages.events.INJECTED_PROVIDER_RPC_RESPONSE.sendToBackgroundPage({
+                        payload,
+                        error: error instanceof Error ? error : new Error(),
+                    })
+                }
+            } else {
+                await new Promise((resolve, reject) =>
```
Comment (guanbinrui) [inline packages/mask/src/plugins/EVM/UI/components/InjectedProviderBridge.tsx:32]: `Promise` is not necessary here.

File: packages/mask/src/extension/background-script/EthereumServices/provider.ts [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import { first } from 'lodash-unified'
+import { defer } from '@masknet/shared-base'
 import type { ChainId, NetworkType, ProviderType } from '@masknet/web3-shared-evm'
 import * as MaskWallet from './providers/MaskWallet'
 import * as MetaMask from './providers/MetaMask'
 import * as WalletConnect from './providers/WalletConnect'
 import * as CustomNetwork from './providers/CustomNetwork'
 import * as Injected from './providers/Injected'
-import { defer } from '@masknet/shared-base'
+import * as Fortmatic from './providers/Fortmatic'
```
Comment (nuanyang233) [inline packages/mask/src/extension/background-script/EthereumServices/provider.ts:9]:
```suggestion
import { requestAccounts } from './providers/Fortmatic'
```

File: packages/mask/src/plugins/EVM/UI/Dashboard/index.tsx [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
 const sns: Plugin.SNSAdaptor.Definition = {
     ...base,
     init(signal) {},
     Web3UI,
     Web3State: {},
+    GlobalInjection() {
+        return (
+            <>
+                <FortmaticProviderBridge />
+            </>
```
Comment (nuanyang233) [inline packages/mask/src/plugins/EVM/UI/Dashboard/index.tsx:15]:
```suggestion
           
                <FortmaticProviderBridge />
           
```

File: packages/mask/src/extension/background-script/EthereumServices/provider.ts (hunk 2 above)
Comment (guanbinrui) [reply packages/mask/src/extension/background-script/EthereumServices/provider.ts:9]: Variable name conflict.

**Pull request merged**